Packs/AbuseDB/ReleaseNotes/1_0_29.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### AbuseIPDB
+- Added the *Disable reputation lookups for private IP addresses* integration parameter to reduce the number of lookups made to the AbuseIPDB API. 
```

Comment:
Change the release notes according to the requested changes please look at this link for xsoar release-notes documentation best practices.
https://xsoar.pan.dev/docs/documentation/release-notes#examples-and-best-practices